### PR TITLE
Courses | duplicate leader issue after rebasing --Resolved By: Umar Waris, Submission Date: June 20, 2016 

### PR DESCRIPTION
**Pull request by umarWaris**

### CODE REVIEW

File: app/MyApp/app/views/GroupForm.js [hunk 1]
```diff
@@ -170,10 +170,10 @@ $(function () {
                 success: function () {
                     //create the form
                     var optns = []
-                   /* optns.push({
-                        label: App.languageDict.attributes.Select_An_option,
-                        val: "0000"
-                    })*/
+                    /* optns.push({
```
Comment (MuhammadWaqas05) [inline app/MyApp/app/views/GroupForm.js]: Commented code should not be pushed/committed. Make sure to delete this code and also remove any un-necassary code whenever you are going to push/commit.

File: app/MyApp/app/views/GroupForm.js [hunk 2]
```diff
@@ -170,10 +170,10 @@ $(function () {
...
+                     label: App.languageDict.attributes.Select_An_option,
```
Comment (SadiaRasheed) [inline app/MyApp/app/views/GroupForm.js]: Code seems fine and can be merged after removing commented code.

**Comment by SadiaRasheed** [comment]
code seem fine and you can merge it.

**Comment by SabaBaig** [comment]
Good to go.

**Pull request merged**